src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -706,17 +795,18 @@ private Result<?, UpdateError> handleAddedTrip(
    * @param realTimeState real-time state of new trip
    * @return empty Result if successful or one containing an error
    */
-  private Result<?, UpdateError> addTripToGraphAndBuffer(
+  private Result<UpdateSuccess, UpdateError> addTripToGraphAndBuffer(
     final Trip trip,
     final TripUpdate tripUpdate,
```

Comment:
Replace with `vehicleDescriptor`, as that is the only thing we need. Now this also contains an outdated list of stop time updates.

Reply:
Great suggestion. Done in f91e087ca95bb7af874a36ff8f2a93acad072124.